MontyTRC89/TombEngine
# Room scripting

Introduces `Room` script object and several script functions for it. Accompanied by a node scripts update on the TE side (see TE repo).

**Things to test:**

* Check if room script functions work normally.
* Check if node script functions for rooms work normally.
* Check if modified flags, reverb type and name remain after savegame reload.
* Check if `GetRoomsByTag` returns correct collection of rooms based on provided tag.
